### PR TITLE
Updates to Provider Boosting Design Docs

### LINKED ISSUE #1318: [Staking Rewards] Economic model design doc
We would like to create an design artifact regarding the Economic model for Staking Rewards.

Acceptance Criteria (partial --sew):
- [ ] presentation outlining options with advantages/disadvantages and how well they meet the requirements
- [ ] choose a model
- [ ] post final PR with chosen model design doc

- tbd @p5150j will flesh out with @shannonwells && @wilwade

### PR DESCRIPTION
**Pull request by shannonwells**
# Goal
The goal of this PR is to add the updates to the Provider Boost Economic Model Design Doc based on the recent economic model decisions by @harry-evans

Closes #1318 

# Discussion
* The implementation and the economic model have been split into two Design Docs.
* Rebased against the implementation Design Doc changes in main and then renamed the file.

# Checklist
- [x] Design doc(s) updated

### CODE REVIEW

**Review by wilwade — COMMENTED** [review]
Few notes on claiming, but everything else aligns with my understanding of the discussions.

File: designdocs/provider_boosting_economic_model.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+2. Bey has locked 1000 FRQCY  (L<sub>u</sub>)  for Provider Boosting.  The total locked by everyone, L<sub>T</sub> for era <i>e</i>,  is 50 Million FRQCY.   The left side of the minimum s `2e6 * 1000 / 50.0e6 = 1000/25 = 40` (that is, 4% of what Bey has locked).  The right side is `1000 * 10% = 100`.  Since 40 is less than 100, Bey's Provider Boost reward is 40 FRQCY.
+
+## Rewards are not issued for a partial Era
+Rewards are not prorated; they are calculated only for balances held for an entire Era.  For example, if an amount is locked at the end of Era 100, and unlocked in Era 101, _no_ Reward will be issued. If an amount is locked in Era 100, and unlocked in Era 102, a Provider Boost Reward is available _only_ for Era 101.
+
+## Claiming Rewards
+* Provider Boost Rewards are not minted until they are explicitly <i>claimed</i> by the Provider Boost account holder, by calling a non-free extrinsic.
+* Rewards must be claimed within a certain number of Provider Boost Eras.
+* When claimed, all available, unexpired Rewards for each previous Era are minted and transferred to the MSA of the Provider Boost Account.
```
[the remark this answers is not in the document]
Comment (shannonwells) [reply designdocs/provider_boosting_economic_model.md]: Ah yes, I forgot.

File: designdocs/provider_boosting_economic_model.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+2. Bey has locked 1000 FRQCY  (L<sub>u</sub>)  for Provider Boosting.  The total locked by everyone, L<sub>T</sub> for era <i>e</i>,  is 50 Million FRQCY.   The left side of the minimum s `2e6 * 1000 / 50.0e6 = 1000/25 = 40` (that is, 4% of what Bey has locked).  The right side is `1000 * 10% = 100`.  Since 40 is less than 100, Bey's Provider Boost reward is 40 FRQCY.
+
+## Rewards are not issued for a partial Era
+Rewards are not prorated; they are calculated only for balances held for an entire Era.  For example, if an amount is locked at the end of Era 100, and unlocked in Era 101, _no_ Reward will be issued. If an amount is locked in Era 100, and unlocked in Era 102, a Provider Boost Reward is available _only_ for Era 101.
+
+## Claiming Rewards
+* Provider Boost Rewards are not minted until they are explicitly <i>claimed</i> by the Provider Boost account holder, by calling a non-free extrinsic.
+* Rewards must be claimed within a certain number of Provider Boost Eras.
+* When claimed, all available, unexpired Rewards for each previous Era are minted and transferred to the MSA of the Provider Boost Account. 
+* **Is there a cap on how much can be claimed at once?**
```
[the remark this answers is not in the document]
Comment (shannonwells) [reply designdocs/provider_boosting_economic_model.md:76]: I can imagine whales staking for the maximum time and then claiming rewards all at once, but I'm not sure what that would do economically.

[the remark this answers is not in the document]
Comment (wilwade) [reply designdocs/provider_boosting_economic_model.md:76]: Even if we capped it, someone could just do it with multiple accounts, but I agree. In theory the economic impact of pending rewards is calculated into peoples mind. (Perhaps a future dashboard should show this number)

File: designdocs/provider_boosting_economic_model.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+2. Bey has locked 1000 FRQCY  (L<sub>u</sub>)  for Provider Boosting.  The total locked by everyone, L<sub>T</sub> for era <i>e</i>,  is 50 Million FRQCY.   The left side of the minimum s `2e6 * 1000 / 50.0e6 = 1000/25 = 40` (that is, 4% of what Bey has locked).  The right side is `1000 * 10% = 100`.  Since 40 is less than 100, Bey's Provider Boost reward is 40 FRQCY.
+
+## Rewards are not issued for a partial Era
+Rewards are not prorated; they are calculated only for balances held for an entire Era.  For example, if an amount is locked at the end of Era 100, and unlocked in Era 101, _no_ Reward will be issued. If an amount is locked in Era 100, and unlocked in Era 102, a Provider Boost Reward is available _only_ for Era 101.
+
+## Claiming Rewards
+* Provider Boost Rewards are not minted until they are explicitly <i>claimed</i> by the Provider Boost account holder, by calling a non-free extrinsic.
```
[the remark this answers is not in the document]
Comment (wilwade) [reply designdocs/provider_boosting_economic_model.md:73]: I'm good with it as is then. We can chose later if claiming requires or doesn't require the holder to do it.

**Review by wilwade — APPROVED** [review]
Looks correct as it can be to me currently.

- ✅ Reviewed

File: designdocs/provider_boosting_economic_model.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+A system consisting only of providers and coinless users who delegate to providers will tend toward centralization.
+To build a self-sustaining Frequency network where control is decentralized, a variety of economic solutions are needed.  One of these is the ability to lock up FRQCY token in return for something; this creates an incentive for participation and involvement with the Frequency chain fundamentals and governance.
+
+How is that so?  Capacity is how Frequency intends Providers to pay for the vast majority of their on-chain messages. In the proposed system, when Providers receive Capacity when end users lock up some FRQCY, they would lose Capacity if those end users unlock. If a Provider's Capacity from Provider Boosting is significant, this gives Provider Boosters some power over their targeted Providers. If a Provider is utilizing all or nearly all their Capacity almost every Epoch -- which they should do if trying to be economical --  then even a small percentage of lost Capacity will literally cost them to replace it.  This gives those end-users relying upon - and Boosting - their Providers the ability to exercise direct market power they did not previously have.
```
Comment (enddynayn) [inline designdocs/provider_boosting_economic_model.md:23]: In the proposed system, ~~when~~ Providers receive Capacity when ~~end~~ users lock up some FRQCY, they ~~would~~ lose Capacity if those ~~end~~ users unlock

File: designdocs/provider_boosting_economic_model.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+End users with Message Source Accounts (MSAs) may receive FRQCY from different sources. Providers may offer airdrops in return for such bringing in new users or sharing links on other platforms, then encourage their users to participate in Provider Boosting.  Rewards could potentially be exchanged for non-transferable, in-app-only benefits such as premium features, special emoji, avatar customization, and the like, similarly to platforms such as [Steam](https://store.steampowered.com).
```
Comment (enddynayn) [inline designdocs/provider_boosting_economic_model.md:25]: Are we able to drop "end users" and say users?

**Review by enddynayn — APPROVED** [review]
👍 exciting!

File: designdocs/provider_boosting_economic_model.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+2. Bey has locked 1000 FRQCY  (L<sub>u</sub>)  for Provider Boosting.  The total locked by everyone, L<sub>T</sub> for era <i>e</i>,  is 50 Million FRQCY.   The left side of the minimum s `2e6 * 1000 / 50.0e6 = 1000/25 = 40` (that is, 4% of what Bey has locked).  The right side is `1000 * 10% = 100`.  Since 40 is less than 100, Bey's Provider Boost reward is 40 FRQCY.
+
+## Rewards are not issued for a partial Era
+Rewards are not prorated; they are calculated only for balances held for an entire Era.  For example, if an amount is locked at the end of Era 100, and unlocked in Era 101, _no_ Reward will be issued. If an amount is locked in Era 100, and unlocked in Era 102, a Provider Boost Reward is available _only_ for Era 101.
+
+## Claiming Rewards
+* Provider Boost Rewards are not minted until they are explicitly <i>claimed</i> by the Provider Boost account holder, by calling a non-free extrinsic.
+* Rewards must be claimed within a certain number of Provider Boost Eras.
```
Comment (saraswatpuneet) [inline designdocs/provider_boosting_economic_model.md:77]: What happens if claimant account did not claim within required numbers of eras?

**Review by saraswatpuneet — APPROVED** [review]
Looks good, nice overview, left one question but not a blocker

File: designdocs/provider_boosting_economic_model.md (hunk 6 above)
Comment (wilwade) [reply designdocs/provider_boosting_economic_model.md:77]: @shannonwells will know for sure, but I believe the oldest rewards are pushed off the stack.

**Pull request merged**